apisix/core/config_etcd.lua

```diff
@@ -568,7 +568,7 @@ local function _automatic_fetch(premature, self)
                     local reconnected = false
                     while err and not reconnected do
                         local backoff_duration, backoff_factor, backoff_step = 1, 2, 10
-                        for _ = 0, backoff_step, 1 do
+                        for _ = 1, backoff_step, 1 do
```

Comment:
We can ignore the pace parameter if it is `1`.

Reply:
Thanks for reminding! Fixed

Reply:
@tokers PTAL